app/models/invitation.rb

```diff
@@ -39,6 +39,7 @@ def accept!
 
   def resend
     notify_invitee unless accepted?
+    update!(sent_at: nil)
   end
```

Comment:
I propose this order:
```suggestion
  def resend
    return if accepted?
    update!(sent_at: nil)
    notify_invitee
  end
```

Mainly because `notify_invitee` updates the `sent_at` attribute in the background at the end of it, and we can not promise that the update of this method will be executed before the other one 😉

Reply:
Yep...